rfcs/rfc-4-text-selection-design.md

```diff
@@ -788,7 +869,7 @@ Accepts any `Color` value.
 
 ## Implementation Guide
```

Comment:
I think the only missing part here would be displaying a cursor

Reply:
Good point. The `Add selection indicators` was meant to cover that but it's not very clear and a little too broad.